.circleci/config.yml

```
@@ -68,6 +68,8 @@ commands:
         type: string
       build_dir:
         type: string
+      build_command:
```

Je trouve le nom du parametre pas terrible.
Pour moi build command ca serait `yarn build` pas just `build`
=> Changer le nom du parametre ou passer `yarn build` en valeur

```suggestion
      yarn_build_subcommand:
```